scripts/build_utils.sh

```diff
@@ -760,6 +760,7 @@ teardown_vagrant_tests() {
         # collect all ceph logs from all test nodes
         collect_ceph_logs all
         vagrant destroy -f
+        stat ./fetch > /dev/null 2>&1 && rm -rf ./fetch
```

Comment:
Why not simply `rm -rf ./fetch`?

Reply:
Indeed, too many sleepless night I guess :-|